bh/documentation/src/badge/examples/BadgeExample.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div>
+    <p><bh-badge color="grey">Lorem</bh-badge> ipsum dolor sit amet,
```

Comment:
Show in this example just badges, create another example to show badge with texts..